### PR TITLE
First step towards pipenv

### PR DESCRIPTION
**Pull request by TimotheusBachinger**
* Use pipenv to isolate python environment
* Makefile with basic target exists

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+include env.make
+
+create-pip-env:
+	pipenv install
```
Comment (a1rw4lk) [inline Makefile]: I have to run `pipenv install --python=python3` on my setup. I'm running under WSL, and at some point pipenv install wants to execute the python.exe under Windows if I don't pass the `--python=python3` argument. Dunno why though, but passing the additional argument doesn't hurt, right?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+setup:
+	sudo -H pip install -U pipenv
```
Comment (a1rw4lk) [inline Makefile:10]: `sudo -H pip3 install -U pipenv`
Or is there a better way to use pip3? Maybe setting it to default in the env.make?

**Review by a1rw4lk — CHANGES_REQUESTED** [review]

**Comment by a1rw4lk** [comment]
Nice work though :D Thanks for contributing <3

File: Makefile (hunk 2 above)
Comment (TimotheusBachinger) [reply Makefile:10]: should not be needed when using pipenv install --python=python3

File: Makefile (hunk 1 above)
Comment (TimotheusBachinger) [reply Makefile]: done

**Pull request merged**